src/commands/initProjectForVSCode/InitVSCodeStep/PythonInitVSCodeStep.ts

```diff
@@ -38,31 +40,35 @@ export class PythonInitVSCodeStep extends ScriptInitVSCodeStep {
     }
 
     protected getTasks(): TaskDefinition[] {
+        const pipInstallLabel: string = 'pipInstall';
+        const dependsOn: string | undefined = this.requiresFuncExtensionsInstall ? extInstallCommand : this._venvName ? pipInstallLabel : undefined;
         const tasks: TaskDefinition[] = [
             {
                 type: func,
                 command: hostStartCommand,
                 problemMatcher: funcWatchProblemMatcher,
                 isBackground: true,
-                dependsOn: extInstallTaskName
+                dependsOn
```

Comment:
If there is an `extInstallCommand` the dependsOn property would just have the venv name, wouldn't it?

Reply:
So I don't know about you, but my diff gets cut off at the edge. It looks like this:
![Screen Shot 2019-04-22 at 3 38 48 PM](https://user-images.githubusercontent.com/11282622/56537197-cb88c400-6514-11e9-8b43-3a78e3bba807.png)

But if you scroll over, I feel like the rest of the line answers your question:
![Screen Shot 2019-04-22 at 3 38 55 PM](https://user-images.githubusercontent.com/11282622/56537216-d4799580-6514-11e9-83f6-5f7640ecd72c.png)

dependsOn will never evaluate to the venvName. Instead it either evaluates to `pipInstallLabel` or undefined